### DISCUSSION TITLE
xmake v2.8.6

### DISCUSSION THREAD

**Discussion by waruqi** [xmake-io/xmake#4539] [Announcements]
### New features

* Add `network.mode` policy
* [#1433](https://github.com/xmake-io/xmake/issues/1433): Add `xmake pack` command to generate NSIS/zip/tar.gz/rpm/srpm/runself packages like cmake/cpack
* [#4435](https://github.com/xmake-io/xmake/issues/4435): Support batchsize for UnityBuild in Group Mode
* [#4485](https://github.com/xmake-io/xmake/pull/4485): Support package.install_locally
* Support NetBSD

### Changes

* [#4484](https://github.com/xmake-io/xmake/pull/4484): Improve swig rule
* Improve Haiku support

### Bugs fixed

* [#4372](https://github.com/xmake-io/xmake/issues/4372): Fix protobuf rules
* [#4439](https://github.com/xmake-io/xmake/issues/4439): Fix asn1c rules

### 新特性

* 添加 `network.mode` 策略
* [#1433](https://github.com/xmake-io/xmake/issues/1433): 添加 `xmake pack` 命令去生成 NSIS/zip/tar.gz/rpm/srpm/runself 安装包
* [#4435](https://github.com/xmake-io/xmake/issues/4435): 为 UnityBuild 的组模式增加 batchsize 支持
* [#4485](https://github.com/xmake-io/xmake/pull/4485): 新增 package.install_locally 策略支持
* 新增 NetBSD 支持

### Changes

* [#4484](https://github.com/xmake-io/xmake/pull/4484): 改进 swig 规则
* 改进 Haiku 支持

### Bugs 修复

* [#4372](https://github.com/xmake-io/xmake/issues/4372): 修复 protobuf 规则
* [#4439](https://github.com/xmake-io/xmake/issues/4439): 修复 asn1c 规则

<hr /><em>This discussion was created from the release <a href='https://github.com/xmake-io/xmake/releases/tag/v2.8.6'>xmake v2.8.6</a>.</em>

**Comment by OpportunityLiu** [comment]
`xmake-v2.8.6.xz.run` is missing

```
root@cloudpss-mini:~# curl -fsSL https://xmake.io/shget.text | bash
xmake, A cross-platform build utility based on Lua.   
Copyright (C) 2015-present Ruki Wang, tboox.org, xmake.io
                         _                            
    __  ___ __  __  __ _| | ______                    
    \ \/ / |  \/  |/ _  | |/ / __ \                   
     >  <  | \__/ | /_| |   <  ___/                   
    /_/\_\_|_|  |_|\__ \|_|\_\____|                   
                         by ruki, xmake.io            
                                                      
   👉  Manual: https://xmake.io/#/getting_started     
   🙏  Donate: https://xmake.io/#/sponsor             
                                                      
downloading https://fastly.jsdelivr.net/gh/xmake-mirror/xmake-releases@v2.8.6/xmake-v2.8.6.xz.run ..
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404 
downloading https://github.com/xmake-io/xmake/releases/download/v2.8.6/xmake-v2.8.6.xz.run ..
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 
make: *** No targets specified and no makefile found.  Stop.
make failed!
```

**Comment by waruqi** [comment]
更新了，再试下，xz 我去了
